### DISCUSSION TITLE
When building from source, Helix won't add itself to path

### DISCUSSION THREAD

**Discussion by itslikeyy** [helix-editor/helix#11420] [General]
### Summary

When building from source, Helix won't add itself to path, so trying to run hx will return an error; user needs to run `export PATH=/home/$USER/src/helix/target/release:$PATH`

We can either add an entry to the wiki about this or make cargo to update the path

### Reproduction Steps

I tried this:

1. build helix from source according to the instructions on the wiki
2. ran hx 

I expected this to happen: Helix opening itself

Instead, this happened: hx: Command not found

### Helix log

<details><summary>~/.cache/helix/helix.log</summary>

Log is empty

</details>

### Platform

Linux

### Terminal Emulator

kitty

### Installation Method

Source

### Helix Version

24.07-105-g0a4432b1

**Comment by pascalkuthe** [comment]
Read the build from source documentation carefully. It instructs using `cargo install --locked`. The latter command copies the executable into the cargo bin directory which is part of the path. Don't use `cargo build`. Build commands never modify the environment only install commands so. That has always been how build tools work and why we don't point at using cargo build in the documentation
